src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -186,6 +189,15 @@ public int getChangedFiles() throws IOException {
     }
 
     /**
+     * FYI: https://developer.github.com/changes/2013-04-25-deprecating-merge-commit-sha
+     */
+    @Deprecated
+    public String getMergeCommitSha() throws IOException {
```

Comment:
When this happens, GitHub API v4 will be released and we should review the current status (compatibility).